src/Runner.Worker/Handlers/OutputManager.cs

```diff
@@ -107,16 +108,15 @@ public void OnDataReceived(object sender, ProcessDataReceivedEventArgs e)
                         // Match
                         try
                         {
-                            match = matcher.Match(stripped);
-
+                            match = matcher.Match(refinedLine);
```

Comment:
The `IssueMatch` returned by `Match` now includes a field that preserves the original string passed to it.